tests/unit/models/addon-test.js

```diff
@@ -278,10 +273,9 @@ describe('models/addon.js', function() {
             },
 
             extensionsForType() {
-              return ['js'];
+              return ['template'];
```

Comment:
This seems odd, why are the test changes required?

Reply:
because `Addon.treeFor` can now return a falsy value where is didn't before. `mergeTrees` helper always returns a truthy tree.